--- lib/cp950ext.h	2021-01-29 02:18:30.000000000 -0700
+++ lib/cp950ext.h	2021-01-29 02:18:40.000000000 -0700
@@ -35,7 +35,7 @@
 cp950ext_mbtowc (conv_t conv, ucs4_t *pwc, const unsigned char *s, size_t n)
 {
   unsigned char c1 = s[0];
-  if ((c1 == 0xf9)) {
+  if  (c1 == 0xf9)  {
     if (n >= 2) {
       unsigned char c2 = s[1];
       if ((c2 >= 0x40 && c2 < 0x7f) || (c2 >= 0xa1 && c2 < 0xff)) {
--- lib/hkscs2001.h.orig	2018-09-17 10:06:43.000000000 -0600
+++ lib/hkscs2001.h	2021-01-29 02:12:48.000000000 -0700
@@ -59,7 +59,7 @@
 hkscs2001_mbtowc (conv_t conv, ucs4_t *pwc, const unsigned char *s, size_t n)
 {
   unsigned char c1 = s[0];
-  if ((c1 == 0x8c)) {
+  if  (c1 == 0x8c)  {
     if (n >= 2) {
       unsigned char c2 = s[1];
       if ((c2 >= 0x40 && c2 < 0x7f) || (c2 >= 0xa1 && c2 < 0xff)) {
--- lib/hkscs2008.h.orig	2021-01-29 02:04:17.000000000 -0700
+++ lib/hkscs2008.h	2021-01-29 01:50:21.000000000 -0700
@@ -55,7 +55,7 @@
 hkscs2008_mbtowc (conv_t conv, ucs4_t *pwc, const unsigned char *s, size_t n)
 {
   unsigned char c1 = s[0];
-  if ((c1 == 0x87)) {
+  if  (c1 == 0x87)  {
     if (n >= 2) {
       unsigned char c2 = s[1];
       if ((c2 >= 0x40 && c2 < 0x7f) || (c2 >= 0xa1 && c2 < 0xff)) {
--- lib/iconv.c.orig	2021-01-29 02:05:58.000000000 -0700
+++ lib/iconv.c	2021-01-29 01:56:41.000000000 -0700
@@ -185,7 +185,7 @@
 inline
 #endif
 #endif
-static const struct alias *
+       const struct alias *
 aliases2_lookup (register const char *str)
 {
   const struct alias * ptr;

